pages/_en-US/gbarunner2/bios-dump.md

```diff
@@ -0,0 +1,81 @@
+---
+lang: en-US
+layout: wiki
+section: gbarunner2
+title: GBA BIOS Dumping
+description: How to extract the Nintendo GBA BIOS from your console
+tabs:
+  - tab-3ds-sd-card: 3DS with open_agb_firm
+    tab-gba-flashcart: GBA/DS/DS Lite with a GBA-mode flashcart
+---
+
+While most GBA emulators have no issue playing GBA games by itself, some emulators and hypervisors, such as GBARunner2, may need the BIOS file to properly playback some titles.
+
+There are two distinct ways to achieve this, using:
+- a 3DS with custom firmware, OR
+- a GBA/DS/DS Lite with a GBA-mode flashcart
+
+{% capture final-steps %}
+### Part 3: Unpack the BIOS from the generated save file
+1. Insert your SD card into your computer
+1. From your SD card, drag `Bios_Dumper.sav` and drop it to `Bios_Shrinker.py`
+1. {% include bios-shrinker.html %}
+    - If you used `open_agb_firm`, this file will be at `/3ds/open_agb_firm/saves` folder
+
+A `bios.bin` file will automatically download. This is the final GBA BIOS file.
+{:.alert .alert-success}
+
+{% endcapture %}
+
+{% capture tab-3ds-sd-card %}
+### Part 1: Getting the required files
+
+1. Download [open_agb_firm](https://github.com/profi200/open_agb_firm/releases/latest) (the `.7z` file)
+1. Download [Bios_Dumper.gba](https://github.com/GlaZedBelmont/Random-Stuff/releases/download/0.0.5/Bios_Dumper.gba)
+1. Create a folder named `payloads` inside the `/luma/` folder if it does not already exist
+1. Copy `open_agb_firm.firm` from the open_agb_firm `.7z` file to the `/luma/payloads` folder
```

Comment:
```suggestion
1. Create a folder named `payloads` inside the `/luma` folder if it does not already exist
1. Copy `open_agb_firm.firm` from the open_agb_firm `.7z` file to the `/luma/payloads` folder
```
inconsistent folder formatting, either use trailing slashes or don't, I think our standard is not

Reply:
I kinda copied this off 3ds guide lol